website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
When you add a component to a node, a log isn't added to the node.  

If I create a project `abcde` and add component `12345`, both nodes have one log each, with action 'project_created' even though `12345` is added to node.nodes.